utils/find-overlay-dependencies.sh

```diff
@@ -14,39 +14,68 @@
 # This script uses zsh for its associative array support, because it is only
 # intended to be run on macOS and macOS bash is very old.
 
-if [[ $# -ne 1 ]]; then
-  echo 'usage:' $0 '<module-name>' >&2
+function usage() {
+  echo 'usage:' $0 '<module-name> [update]' >&2
```

Comment:
Any reason not to squash the loop script into this one? (invoking itself with `$0`). It seems tidier for utils/.